code/lib/types/src/modules/core-common.ts

```diff
@@ -156,6 +157,7 @@ export interface CLIOptions {
   quiet?: boolean;
   versionUpdates?: boolean;
   docs?: boolean;
+  test?: boolean;
```

Comment:
`test` feels a little bit ambiguous to me.  I can imagine a new user looking at these options and wondering what the difference is between `test` and `ci`, for example.  At the risk of being verbose, I would suggest `testMode` or `testingMode` or just `testing` or to be symmetric with `docs`: `tests`.

Reply:
I changed it to `build.test`

Reply:
@IanVS We had a long internal discussion about this. I'm okay with `--test-mode` or `--testing`, but I guess if we have good docs `--test` should be fine as well.

Reply:
Sure, though I don't like relying on docs to make up for confusing names.  Not everyone reads docs, and the command may be put into a package.json script where the docs are not at hand when someone is reading it and trying to understand what it does.  I kind of like `--tests`, to match `--docs`, personally, since they are both special modes of running Storybook, it feels like they should kind of match up.

That said, I don't feel super-strongly about the naming, just wanted to bring it up as a potentially confusing name.

Reply:
@shilman What do you think about —tests?